README.md

```diff
@@ -1,12 +1,98 @@
+absence of certain characteristics of the property/area using a simple additive model (see table below). Scores are then
+averaged per property before finally aggregating to LSOA level.
+
+|                                                                                   | ASHP (S) | ASHP (N) | GSHP (S) | GSHP (N) | SGL (S) | SGL (N) | HN (S) | HN (N) |
+| --------------------------------------------------------------------------------- | -------- | -------- | -------- | -------- | ------- | ------- | ------ | ------ |
+| Is the property NOT listed?                                                       | 0.25     | 0.25     | 0.25     | 0.25     | 0.25    | 0.25    | 0.25   | 0.25   |
+| Is the property NOT in a building conservation zone?                              | 0.25     | 0.25     | 0.25     | 0.25     | 0.25    | 0.25    | 0.25   | 0.25   |
+| Is the EPC rating >= C?                                                           | 1        | 0        | 1        | 0        | 1       | 0       | 0      | 0      |
+| Is the property NOT a flat?                                                       | 1        | 1        | 1        | 1        | 0       | 0       | 0      | 0      |
+| Is the garden >10m2?                                                              | 1        | 0        | 1        | 0        | 1       | 0       | 0      | 0      |
+| Is there >2m2 of external space?                                                  | 0        | 2        | 0        | 1        | 0       | 0       | 0      | 0      |
+| Is it off-gas?                                                                    | 0.5      | 0.5      | 0.5      | 0.5      | 0.5     | 0.5     | 0.5    | 0.5    |
+| Is this property part of a building with multiple other properties (e.g. a flat)? | 0        | 0        | 0        | 0        | 2       | 2       | 2      | 2      |
+| Is there a high property density (>60 households per km2) in this LSOA?           | 0        | 0        | 0        | 0        | 2       | 2       | 0      | 0      |
+| Is this property in an urban LSOA/high heat demand density LSOA?                  | 0        | 0        | 0        | 0        | 0       | 0       | 2      | 2      |
+| Maximum points per property                                                       | 4        | 4        | 4        | 3        | 7       | 5       | 5      | 5      |
+
+## Data sources and acknowledgements
+
+A comprehensive table of citations for data used in this analysis can be found in [asf_heat_pump_suitability/config/README.md](https://github.com/nestauk/asf_heat_pump_suitability/tree/dev/asf_heat_pump_suitability/config#readme).
+
+Contains OS data © Crown copyright and database right 2023.\
+Contains Royal Mail data © Royal Mail copyright and database right 2023.\
+Contains Office for National Statistics information licensed under the Open Government Licence v.3.0.\
+Contains public sector information licensed under the Open Government Licence v3.0.
+
```

Comment:
Add the following information for the land registry:

This work uses HM Land Registry's INSPIRE Index Polygons service. This information is subject to Crown copyright and database rights 2024 and is reproduced with the permission of HM Land Registry.

The polygons (including the associated geometry, namely x, y co-ordinates) are subject to Crown copyright and database rights 2024 Ordnance Survey 100026316.

INSPIRE Index Polygons service [Conditions of use](https://use-land-property-data.service.gov.uk/datasets/inspire#conditions)